pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -35,6 +35,8 @@
     <ExcludeFromClosure Include="System.Net" />
     <ExcludeFromClosure Include="System.Transactions" />
 
+    <ExcludeFromDuplicateTypes Include="System.Private.Reflection.Metadata.Ecma335" />
```

Comment:
This has type overlap with much of S.R.Metadata.  /cc @tmat @davidwrighton 

I wonder if we actually need to give this a different name?  We have a similar problem with this assembly in .NETCoreApp.  We cannot use the version of the assembly that builds against netstandard.dll because we need to use this assembly in the implementation of other assemblies which make up netstandard.dll.

It seems like this is a similar problem expect it needs to go even lower.  Did you guys consider having this thing keep its regular name and just have it compile against S.P.C on UAP?